src/networking.c

```diff
@@ -654,22 +654,22 @@ void addReplyErrorExpireTime(client *c) {
                         c->cmd->fullname);
 }
 
-void addReplyStatusLength(client *c, const char *s, size_t len) {
+void addReplySimpleString(client *c, const char *s, size_t len) {
```

Comment:
i know the RESP2 documentation refers to these as "simple strings" but i the source code they're refereed to as "status", and changing just one of them looks odd (i.e. we still have `addReplyStatus` and `addReplyStatusFormat`, so why rename `addReplyStatusLength`)

i'd rather revert that.

Reply:
reverted it in the latest commit =)